### PR TITLE
Use upper case

### PR DESCRIPTION
**Pull request by jbleduigou**
Use upper case for default values when asking user for confirmation 
https://github.com/kyma-project/cli/issues/255

### CODE REVIEW

File: internal/root/root.go [hunk 1]
```diff
@@ -4,13 +4,13 @@ import "fmt"
...
 	for {
-		fmt.Print("Type [y/n]: ")
+		fmt.Print("Type [Y/n]: ")
 		var res string
 		if _, err := fmt.Scanf("%s", &res); err != nil {
 			return false
 		}
 		switch res {
-		case "yes", "y":
+		case "Yes", "Y":
```
Comment (clebs) [inline internal/root/root.go]: The yes option should not only accept a capital Y, people are used to just press y + enter.
Please could you add also lower case `y` and `yes`? Thanks!

**Comment by clebs** [comment]
Hi @jbleduigou,

thanks for your Pull Request!

We just migrated the Kyma CLI to `go modules`. Would you be so kind to rebase your branch with the latest changes from master? Otherwise the CI/CD will fail.

Thanks and sorry for the inconvenience!

File: internal/root/root.go (hunk 1 above)
Comment (jbleduigou) [reply internal/root/root.go]: Yes indeed, work was done

**Pull request merged**